tests/sentry/integrations/github/test_issues.py

```diff
@@ -77,48 +87,86 @@ def test_get_allowed_assignees(self, mock_get_jwt):
             self._check_proxying()
 
     @responses.activate
-    @patch("sentry.integrations.github.client.get_jwt", return_value="jwt_token_1")
-    def test_get_repo_labels(self, mock_get_jwt):
+    @patch.object(sentry.integrations.github.client.GitHubClientMixin, "page_size", 1)
+    def test_get_labels_all_and_pagination(self):
```

Comment:
I copied a lot of the pagination logic from testing the paginated get repositories test here
https://github.com/getsentry/sentry/blob/master/tests/sentry/integrations/github/test_integration.py#L149-L185